andrey-burban/Homepage/css/style.css

```diff
@@ -0,0 +1,459 @@
+*{
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+body{
+  width: 1260px;
+  height: 1223px;
+  font-family: 'Oswald', sans-serif;
+  background-image: url(../images/background.png);
+}
+
+.wrapper{
+  width: 960px;
+  margin: 0 auto;
+}
+
+ .top-line{
+  width: 1260px;
+  height: 5px;
+  background-image: url(../images/top_line.png);
+}
+
+/*HEADER*/
+
+  header{
+  position: relative;
+  padding-top: 25px;
+  margin-bottom: 32px;
+}
+
+  header:after{
+  content:'';
+  display: table;
+  clear: both;
+}
+
+.logo{
+  float: left;
+}
+
+.search-field {
+  float: right;
+}
+
+input{
+  width: 157px;
+  height: 30px;
+  margin-right: 42px;
+  background-color: #f3f3f3;
+  border: 1px solid #e7e7e7;
+}
+
+.go-button{
+  position: absolute;
```

Comment:
fixed without positioning, input "search" and "submit" added, changed method from post to get